templates/gsheets_salesforce_integration/src/gsheets_salesforce_integration/Module.md

```diff
@@ -0,0 +1,58 @@
+# Google spreadsheet and Salesforce API Integration
+
+This template demonstrates some of the capabilities of Google spreadsheet and Salesforce modules.
+
+`salesforceService` HTTP service will be created with this template. salesforceService has a 3 resources called 
+`addAccount`, `addContacts` and `deleteContacts`. `addAccount` resource creates a Account in Salesforce using a given 
+JSON. `addContacts` resource creates contacts in Salesforce for a given Account using Google spreadsheet data. Here
+we are going to add contacts in 
+[this spreadsheet](https://docs.google.com/spreadsheets/d/1nROELRHZ9JadnvIBizBfnx0FASo2tg7r-gRP1ribYNY/edit?usp=sharing) 
+to Salesforce. `deleteContacts` resource deletes Salesforce contacts for a given Account.
+
+## How to run the template
+
+1. Alter the config file `src/gsheets_salesforce_integration/resources/ballerina.conf` as per the requirement.
+
+2.  Execute following command to run the service.
+    ```bash
+    ballerina run xml_to_json_transformation --config src/xml_to_json_transformation/resources/ballerina.conf
```

Comment:
Ballerina run command seems to be incorrect.

Reply:
Fixed in https://github.com/wso2/ballerina-integrator/pull/237